app/assets/javascripts/discourse/app/components/login-buttons.js

```diff
@@ -16,6 +17,11 @@ export default Component.extend({
     return findAll();
   },
 
+  @discourseComputed
+  canUsePasskeys() {
+    return isWebauthnSupported() && this.siteSettings.experimental_passkeys;
```

Comment:
```suggestion
    return this.siteSettings.experimental_passkeys && isWebauthnSupported();
```

Flipping the order around since calling out to site settings is likely cheaper than checking for `PublicKeyCredential`